dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/constants/ApiFuncIdentificationConstant.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.api.constants;
+
+import org.apache.dolphinscheduler.api.enums.ExecuteType;
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class ApiFuncIdentificationConstant {
+
+    public static final String ACCESS_TOKEN_MANAGE = "security:token:view";
```

Comment:
I think we should use prefix "PERMISSION_CHECK" to identify that this variable is permission check, but this is not important, we can fix it later.

Reply:
ok.